aio_pika/exceptions.py

```diff
@@ -11,9 +11,7 @@
 )
 
 
-PAMQP_EXCEPTIONS = (pamqp.exceptions.PAMQPException,) + tuple(
-    pamqp.specification.ERRORS.values(),
-)
+PAMQP_EXCEPTIONS = (pamqp.exceptions.PAMQPException,)
 
 CONNECTION_EXCEPTIONS = (
```

Comment:
I think `ConnectionError` can be removed because it inherits `OSError` which is already exists here.